webclient/app/src/app/commons/alert/ErrorAlert.jsx

```diff
@@ -0,0 +1,22 @@
+import {
```

Comment:
Maybe a more integrated approach for error handling could be a bit more user-friendly than a banner. Maybe we should talk about that one :)

Reply:
there shouldn't be an error message if validation is implemented. I am waiting for it from EditDialog.

Reply:
From the user input side, I fully agree, but there could always be errors during pushing data to the server or server-internal errors.